projects/io-backend-projects/provider.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  backend "azurerm" {
```

Comment:
Since you are not using terragrunt ...what about giving a try with terraform cloud?

Reply:
not blocking ... up to you !!!

Reply:
Absolutely, but I'd like to complete this PoC with all others tasks before start to use it.